In [1]:
import numpy as np
import scipy.stats as stats

In [2]:
x_bar = 3050        # sample mean
mu = 4000           # theoretical mean
sigma = 125         # population standard deviation
n = 25              # sample size

In [3]:
z = (x_bar - mu) / (sigma / np.sqrt(n))
z

np.float64(-38.0)

In [4]:
p_value = 1 - stats.norm.cdf(z)
p_value

np.float64(1.0)

In [5]:
z_critical = stats.norm.ppf(1 - 0.05)
z_critical

np.float64(1.6448536269514722)

##### −38<1.645⇒Do NOT reject H0

##### The observed sample mean weekly cost is significantly lower than the theoretical cost of ₹4000. Therefore, the cost model does not underestimate weekly operating costs.